connector/connect/client/jvm/pom.xml

```diff
@@ -105,11 +88,16 @@
           <artifactSet>
             <includes>
               <include>com.google.guava:*</include>
+              <include>io.grpc:*:</include>
```

Comment:
should revert this change after https://github.com/apache/spark/pull/39789 merged